.github/workflows/build.yml

```diff
@@ -27,6 +27,12 @@ jobs:
       ARTIFACTS_DIR: exported-artifacts/${{ matrix.name }}
 
     steps:
+    - name: Install git package
+      # git needs to be installed before checkout action invocation, otherwise
+      # the action won't fetch git details correctly
+      run: |
+        dnf install -y git
+
     - name: Checkout sources
       uses: actions/checkout@v2
 
```

Comment:
```
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

Reply:
fetch-depth: 0 is not necessary if we want only use git rev-parse and git archive, it works fine even when only the last commit is present